corehq/apps/translations/generators.py

```diff
@@ -250,14 +250,14 @@ def _get_translation_for_sheet(self, app, sheet_name, rows):
         default_lang_index = self._get_header_index(sheet_name, self.lang_prefix + app.default_language)
         if sheet_name == MODULES_AND_FORMS_SHEET_NAME:
             type_index = self._get_header_index(MODULES_AND_FORMS_SHEET_NAME, 'Type')
-            sheet_name_index = self._get_header_index(MODULES_AND_FORMS_SHEET_NAME, 'sheet_name')
+            sheet_name_index = self._get_header_index(MODULES_AND_FORMS_SHEET_NAME, 'menu_or_form')
             unique_id_index = self._get_header_index(MODULES_AND_FORMS_SHEET_NAME, 'unique_id')
 
             def occurrence(_row):
                 return ':'.join([_row[type_index], _row[sheet_name_index], _row[unique_id_index]])
```

Comment:
This line currently generates this `Module:module1:ad7059becbdcd3dfe155c116d5ac6837017b4e31` but after this change it would generate `Menu:menu1:ad7059becbdcd3dfe155c116d5ac6837017b4e31`. We would want to keep it `Module` and `module` only because a change in occurrence would cause the existing translations to get removed.

Reply:
thank you - https://github.com/dimagi/commcare-hq/pull/23924